test/integration/source_openssl/openssl_spec.rb

```diff
@@ -0,0 +1,11 @@
+describe directory '/usr/local/ssl/' do
+  it { should exist }
+end
+
+describe command('openssl version -a | grep OpenSSL') do
+  its('stdout') { should match /OpenSSL 1.1.1h/ }
+end
+
+describe command('haproxy -vv | grep OpenSSL') do
+  its('stdout') { should match /Built with OpenSSL version/ }
```

Comment:
I recommend you update this to the following and remove the openssl command above. This will ensure we got the correct version.
```suggestion
  its('stdout') { should match /Built with OpenSSL version : OpenSSL 1.1.1h  22 Sep 2020/ }
```